srcs/parser/parser_color_texture.c

```diff
@@ -73,19 +73,34 @@ int	texture_is_valid(t_cub *cub, t_parse_info *parse_info, char *content)
 		return (0);
 	if ((fd = open(parse_info->file_name, O_RDONLY)) == -1)
 		return (error_and_return(FILE_INEXISTENT, 0));
-	if ((ft_strncmp(parse_info->prefix, "NO", 2) == 0 && cub->no_fd != 0) ||
-		(ft_strncmp(parse_info->prefix, "SO", 2) == 0 && cub->so_fd != 0) ||
-		(ft_strncmp(parse_info->prefix, "WE", 2) == 0 && cub->we_fd != 0) ||
-		(ft_strncmp(parse_info->prefix, "EA", 2) == 0 && cub->ea_fd != 0))
+	if ((ft_strncmp(parse_info->prefix, "NO", 2) == 0 && cub->walls.no_fd != 0) ||
```

Comment:
if we remove the fds (which we should) we should initialize the strings for the filenames to NULL, and in here check if the names are != NULL (instead of checking if the fds != 0)

Reply:
but I can do this refactor after you're done with the textures if you want